dataloader/dataloader_utils.py

```diff
@@ -84,13 +84,13 @@ def should_download_file(file_lang: str, file_name: str) -> bool:
     Limit source file set size to speed up loading process
     Can be controlled with the `LIMIT` environment variable.
     """
-    if file_lang == LANG_CHINESE:
+    if file_lang == LANG_CHINESE and not "_0220" in file_name:
```

Comment:
This is leftover from debugging but will leave it for now. It changes so often. I often forget before I push too.